.github/workflows/firefox-ios-sync-acorn-icons.yml

```diff
@@ -0,0 +1,52 @@
+name: Sync Icons from Acorn repo
+
+on: 
+    workflow_dispatch:
+    schedule:
+      - cron: '0 12 * * SUN'
+
+jobs:
+    download_icons_if_needed:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+    
+            - name: Set up Python
+              uses: actions/setup-python@v4
+              with:
+               python-version: '3.x'
+    
+            - name: Install dependencies
+              run: |
+               python -m pip install --upgrade pip
+               pip install requests
+    
+            - name: Download Acorn icons
+              run: |
+               python sync_acorn_icons.py
+            
+            - name: Check changes
+              run: |
+                if [[ $(git status --porcelain) ]]; then
+                    echo "changes_detected=true" >> $GITHUB_ENV 
+                fi
+
+            - name: Commit if needed 
+              if: ${{ env.changes_detected }}
+              run: |
+                git config --global user.name "github-actions[bot]"
+                git config --global user.email "github-actions[bot]@users.noreply.github.com"
+                git add .
```

Comment:
I usually prefer to specify what to add... otherwise there might be other unexpected files added to the PR... see for example : https://github.com/mozilla-mobile/firefox-ios/blob/eb53b791f7243dfd3968b5130376c186ed3995e8/.github/workflows/check-rust-component-dependency.yml#L72

Reply:
Ok perfect thank you Isabel i'm going to change it